cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/StopStartUpscaleStartInstancesHandler.java

```diff
@@ -51,7 +51,6 @@ public void accept(Event<StopStartUpscaleStartInstancesRequest> event) {
 
         int numInstancesToStart = request.getNumInstancesToStart() - request.getStartedInstancesWithServicesNotRunning().size();
         if (numInstancesToStart <= 0) {
-            // TODO CB-15132: This is currently not exercised, since the Flow itself does not send any good information about running instances.
```

Comment:
If this is negative, how many nodes should be used in the next step during commissioning? If upscale request is for 5 nodes, and there's 10 such nodes in the system - do we go ahead and commission all 10, or only 5.

Also, if a downscale was invoked first - would we have ended up STOPPING the 10 nodes on which services were not running?